NEWS.md

```diff
@@ -1,5 +1,7 @@
 <h2>News</h2>
 
+2017-06-21: [MTurk now supports multiple assignments per HIT](https://github.com/facebookresearch/ParlAI/pull/156)
```

Comment:
also general worlds and agents inside them and junk no?  (tutorial updated too, right?)